programs/client/Client.cpp

```diff
@@ -1074,8 +1111,11 @@ void Client::processOptions(const OptionsDescription & options_description,
 
     if (options.count("config"))
         config().setString("config-file", options["config"].as<std::string>());
-    if (options.count("host") && !options["host"].defaulted())
-        config().setString("host", options["host"].as<std::string>());
+    if (options.count("host"))
+    {
+        hosts_ports = options["host"].as<std::vector<HostPort>>();
+        config().setString("host", hosts_ports[0].host);
```

Comment:
You set it here, but anyway you do it later in a round robin loop. Do you really need it here?

Reply:
I need to set "host" before round-robin, because I need to create `ConnectionParameters(config())` in `connect` function. I thought, that it would be better to preset `config()` before calling `connect`.